dadi/lib/controller/index.js

```diff
@@ -296,34 +296,27 @@ Controller.prototype.loadEventData = function (events, req, res, data, done) {
 }
 
 Controller.prototype.loadData = function (req, res, data, done) {
-  var self = this
+  const self = this
 
-  var primaryDatasources = {}
-  var chainedDatasources = {}
+  let primaryDatasources = {}
+  let chainedDatasources = {}
```

Comment:
Do these two variables ever get reassigned away from the `{}` object they begin with? If it's only the object they reference that mutates, we can still use `const` here.

Reply:
true

Reply:
done